app/conversations/[conversationId]/components/ProfileDrawer.tsx

```diff
@@ -18,6 +19,7 @@ interface ProfileDrawerProps {
 
 const ProfileDrawer: FC<ProfileDrawerProps> = ({ isOpen, data, onClose }) => {
     const otherUser = useOtherUser(data)
+    const [isModalOpen, setIsModalOpen] = useState(false)
```

Comment:
change the isModalOpen to isConfirmOpen in all your file and the setIsModalOpen to setIsConfirmOpen in all your file too